spec/features/admin_claim_check_spec.rb

```diff
@@ -80,6 +80,15 @@
       expect(page).to have_content(user.full_name)
     end
 
+    context "When the claim has had PII removed" do
+      let!(:claim_with_pii_removed) { create(:claim, :rejected, :pii_removed) }
+
+      scenario "User can see when a claim has had PII removed" do
+        visit admin_claim_path(claim_with_pii_removed)
+        expect(page).to have_content("personally identifiable information removed")
```

Comment:
Can we also verify that at least one of the scrubbed fields is listed as `"Removed"` .